src/dynapi/SDL_dynapi_procs.h

```diff
@@ -84,7 +84,7 @@ SDL_DYNAPI_PROC(int,SDL_LinuxSetThreadPriority,(Sint64 a, int b),(a,b),return)
 SDL_DYNAPI_PROC(int,SDL_LinuxSetThreadPriorityAndPolicy,(Sint64 a, int b, int c),(a,b,c),return)
 
 SDL_DYNAPI_PROC(void,SDL_OnApplicationDidChangeStatusBarOrientation,(void),(),)
-SDL_DYNAPI_PROC(int,SDL_iPhoneSetAnimationCallback,(SDL_Window *a, int b, void (SDLCALL *c)(void *), void *d),(a,b,c,d),return)
```

Comment:
You can revert this commit. The whitespace after your regeneration is correct.

Reply:
You mean the before is correct? So `void (SDLCALL *c)(void *)`?

What about `SDL_GamepadBinding**`/`SDL_GamepadBinding **`?

Reply:
Yes, there should be space between `SDL_GamepadBinding` and `**`

Reply:
I've fixed the whitespace of `SDL_iPhoneSetAnimationCallback` in the original header file as the python script uses that.
But the script doesn't respect the whitespace in the return type `SDL_GamepadBinding **`.

Reply:
It's not super important, you can leave it however the script generates it.